roles/splunk_common/tasks/java_tasks/install_oracle8_jdk.yml

```diff
@@ -1,9 +1,9 @@
 ---
 - name: Download the Java distribution
   get_url:
-    url: https://download.oracle.com/otn-pub/java/jdk/8u191-b12/2787e4a523244c269598db4e85c51e0c/jdk-8u191-linux-x64.tar.gz
+    url: "{{ java_oracle_url }}"
```

Comment:
Wonder if we should add something like:
```
retries: 5
delay: 10
``` 
to this section? We may be more failure-tolerant to ephemeral networking problems this way.

Reply:
Maybe also add a `timeout` into the get_url parameter: https://docs.ansible.com/ansible/latest/modules/get_url_module.html

The default is 10s, which might not be conservative enough. Maybe something like 30s would be safe.